src/airport.js

```diff
@@ -33,12 +34,15 @@ class Airport {
     
     
   }
+
   isAtCapacity(){
```

Comment:
I would like to use the method isAtCapacity in the landPlane method instead of checking again the condition "this.airportPlanes.length>= this.capacity". This would require that the isAtCapacity method returns a "true" or "false" boolean instead of a string, and maybe use a console.log to display that the airport is full when that's the case.